src/Analyzer.Utilities/Extensions/IEnumerableOfIMethodSymbolExtensions.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+
+namespace Analyzer.Utilities
+{
+    public static class IEnumerableOfIMethodSymbolExtensions
+    {
+        /// <summary>
+        /// Returns a list of method symbols from a given list of the method symbols, which has its parameter type as
```

Comment:
Yes, this is used in the analyzer `SpecifyIFormatProvider.cs` for which I have a PR out #912. This helper will be shared between them